pkg/cert/cert.go

```diff
@@ -57,7 +61,12 @@ func Info(pemCerts, pemKey string) (*CertificateInfo, error) {
 			continue
 		}
 
-		if pubKey.N.Cmp(key.N) != 0 {
+		privKey, ok := key.(*rsa.PrivateKey)
```

Comment:
What's the logic in adding this check?  Other kinds of keys (elliptical in particular) are valid too and nothing in the private key structure is used (except N, which is == to pubKey.N)

Reply:
It's needed because `ParsePKCS8PrivateKey` returns an interface, and we use key specific properties later (N).

```
func ParsePKCS1PrivateKey(der []byte) (*rsa.PrivateKey, error)
vs
func ParsePKCS8PrivateKey(der []byte) (key interface{}, err error)
```

Let me add EC as well
```
func ParseECPrivateKey(der []byte) (*ecdsa.PrivateKey, error)
```

Reply:
@deniseschannon @vincent99 do we want to merge PKCS8 support or close and have EC added in a new PR?

Reply:
I guess there's no such thing as EC in PKCS#1, so merging this with support for only RSA in PKCS#8 and adding EC later is ok.